.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+#  @department-of-veterans-affairs/backend-review-group will be requested for
+# review when someone opens a pull request.
+
+*       @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/ad-hoc-devops
+
+*.md    @department-of-veterans-affairs/teams/ad-hoc-general @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/ad-hoc-devops
```

Comment:
@annaswims do you have background on why `*.md` has a different precedence order?

Reply:
I audited PR's from the last month and there was one PR that was documentation only that was approved by a product person. But on second thought it's not that hard to get a 👍 from a dev in one of these groups so I think I'll just take out this line.